jvm-packages/create_jni.py

```diff
@@ -18,7 +18,7 @@
     "USE_HDFS": "OFF",
     "USE_AZURE": "OFF",
     "USE_S3": "OFF",
-
+    "RABIT_MOCK": "OFF",
```

Comment:
is it harmful to include RABIT_MOCK.so in rabit build directly?

Reply:
removed.